src/components/accordion/__tests__/CdrAccordionGroup.spec.js

```diff
@@ -3,79 +3,95 @@ import CdrAccordionGroup from '../CdrAccordionGroup.vue';
 import CdrAccordion from '../CdrAccordion.vue';
 import { h } from '@vue/runtime-core';
 
-const propsData = {
-  id: 'test',
-  label: 'A label',
-  compact: false,
-  borderAligned: false,
-};
-
 describe('CdrAccordionGroup', () => {
-  it('renders correctly', () => {
-    const wrapper = mount(h(CdrAccordionGroup, {}, {default: () => [
-      h(CdrAccordion, {id: 'tab1', level: '2', label: 'label1'}),
-      h(CdrAccordion, {id: 'tab2', level: '2', label: 'label2'}),
-    ]}));
-    expect(wrapper.element).toMatchSnapshot()
-  });
+  let wrapper;
 
-  it('renders correctly unwrapped', () => {
-    const wrapper = mount(h(CdrAccordionGroup, { unwrap: true }, {default: () => [
-      h(CdrAccordion, {id: 'tab1', level: '2', label: 'label1'}),
-      h(CdrAccordion, {id: 'tab2', level: '2', label: 'label2'}),
-    ]}));
-    expect(wrapper.element).toMatchSnapshot()
+  describe('when mounted', () => {
+    beforeEach(()=>{
+      wrapper = mount(h(CdrAccordionGroup, {}, {default: () => [
+        h(CdrAccordion, {id: 'tab1', level: '2', label: 'label1'}),
+        h(CdrAccordion, {id: 'tab2', level: '2', label: 'label2'}),
+      ]}));
+    })
+
+    it('renders correctly', () => {
+      expect(wrapper.element).toMatchSnapshot()
+    });  
   });
 
-  it('has correct a11y', async () => {
-    // const elem = document.createElement('div')
-    // if (document.body) {
-    //   document.body.appendChild(elem)
-    // }
-    //   attachTo: elem,
+  describe('when unwrapped', () => {
```

Comment:
@benjag your comment on the other describe makes me rethink this describe as well. Maybe we should separate our snapshot tests from our unit tests? In that case, I'd move the one test for a11y requirements into its own describe and mount the same component. Feels a bit redundant but may be good for separation of concerns